### PR TITLE
Meta/print odp view

### PR DESCRIPTION
**Pull request by sorja**
Part of #1433 
- [x] OriginalDataPointsPrint

### CODE REVIEW

File: common/api/endpoint/ApiEndPoint.ts [hunk 1]
```diff
@@ -43,6 +43,10 @@ export const ApiEndPoint = {
     OriginalDataPoint: {
       one: (countryIso = ':countryIso', assessmentName = ':assessmentName', cycleName = ':cycleName', year = ':year') =>
         apiPath('country', countryIso, 'assessment', assessmentName, 'originalDataPoint', cycleName, year),
+
+      many: (countryIso = ':countryIso', assessmentName = ':assessmentName', cycleName = ':cycleName') =>
```
Comment (sorja) [inline common/api/endpoint/ApiEndPoint.ts:47]: Done with old convention to be in line with previous (`one`) for easier refactor later

File: client/pages/AssessmentPrint/OriginalDataPointsPrint/useOriginalDataPoints.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+export const useOriginalDataPoints = (): { originalDataPoints: Array<OriginalDataPoint>; loading: boolean } => {
+  const assessment = useAssessment()
+  const cycle = useCycle()
+  const countryIso = useCountryIso()
+
+  const {
+    data: originalDataPoints = [],
+    dispatch: fetchResults,
+    loading,
+  } = useGetRequest(ApiEndPoint.Assessment.OriginalDataPoint.many(countryIso, assessment.props.name, cycle.name))
```
Comment (minotogna) [inline client/pages/AssessmentPrint/OriginalDataPointsPrint/useOriginalDataPoints.ts:19]: dont we get the odp when fetching data already ?

Comment (minotogna) [reply client/pages/AssessmentPrint/OriginalDataPointsPrint/useOriginalDataPoints.ts:19]: we need all of it I guess..

File: server/repository/assessmentCycle/originalDataPoint/getMany.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { Objects } from '@core/utils'
+
+import { CountryIso } from '@meta/area'
+import { Assessment, Cycle, OriginalDataPoint } from '@meta/assessment'
+
+import { BaseProtocol, DB, Schemas } from '@server/db'
+
+export const getMany = async (
+  props: { assessment: Assessment; assessmentCycle: Cycle; countryIso: CountryIso },
```
Comment (minotogna) [inline server/repository/assessmentCycle/originalDataPoint/getMany.ts:9]: call this parameter cycle

**Review by minotogna — APPROVED** [review]

**Pull request merged**